students/mielczarek_szymon/lesson_02_flow_control/the_number_of_zeros.py

```diff
@@ -0,0 +1,8 @@
+n = int(input())
+cnt = 0
+
+for i in range(n):
```

Comment:
"i" variable is not used in this context.
by convetion this should be changed to underscore '_'

"As a general purpose "throwaway" variable name to indicate that part of a function result is being deliberately ignored"
from:
https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python